.travis.yml

```diff
@@ -46,51 +46,6 @@ matrix:
       distributions: sdist
       on:
         tags: true
-  - script: scripts/test-build.sh
-    install:
-    - pip install --user pybind11 nose2
-    cache: ccache
-    env:
-    - BUILD_JOB=Build in RelWithDebInfo mode
-    - TRAVIS_BUILD_TYPE=RelWithDebInfo
-    addons:
-      apt:
-        packages:
-        - libjson-c-dev
-        - libhwloc-dev
-        - libtbb-dev
-        - uuid-dev
-        - doxygen
-  - script: scripts/test-build.sh
-    install:
-    - pip install --user pybind11 nose2
-    cache: ccache
-    env:
-    - BUILD_JOB=Build in Release mode
-    - TRAVIS_BUILD_TYPE=Release
-    addons:
-      apt:
-        packages:
-        - libjson-c-dev
-        - libhwloc-dev
-        - libtbb-dev
-        - uuid-dev
-        - doxygen
-  - script: scripts/test-build.sh
```

Comment:
@r-rojo will this change depreciate the `scripts/test-build.sh` script?

Reply:
If so, could/should it be removed?

Reply:
Yes, and yes